electron/preload.js

```diff
@@ -94,7 +94,7 @@ async function generateLndConfigFromWallet(wallet) {
   // Wait for the config to become fully initialized.
   await lndConfig.isReady
 
-  return lndConfig
+  return Object.assign({}, lndConfig)
```

Comment:
Could you add a comment here, please? I assume we just create an independent copy of config here, so the original one stays untouched no matter what?

Reply:
Have added some comments to this.